packages/dotlottie-js/src/common/lottie-image-common.ts

```diff
@@ -140,6 +140,8 @@ export class LottieImageCommon {
       }
       // Default to png if the file extension isn't available
       this.fileName = `${newName}.${fileExt}`;
+    } else {
+      this.fileName = `${newName}.png`;
```

Comment:
Isn't the extension inferred from the mimetype of the data ? defaulting it to png is safe ?